string objList
string objListAtoms
uint32 index
---
string result
